src/pseudo/NormConservingPsp.jl

```diff
@@ -48,9 +53,10 @@ Evaluate the local part of the pseudopotential in reciprocal space:
 V(q) = ∫_R^3 Vloc(r) e^{-iqr} dr
      = 4π ∫_{R+} sin(qr)/q r e^{-iqr} dr
 """
-eval_psp_local_fourier(psp::NormConservingPsp, q::T) where {T <: Real} = 
+eval_psp_local_fourier(psp::NormConservingPsp, q) =
```

Comment:
Not sure it matters, but here you dropped the `::Real`. I'd add it for consistency.

Reply:
woops you're right, thanks